### PR TITLE
Added ability to use Sphinx with markdown.

### PR DESCRIPTION
**Pull request by JimiC**
Resolves issue #74

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+# If true, do not generate a @detailmenu in the "Top" node's menu.
+#texinfo_no_detailmenu = False
+
+# Markdown support
+source_parsers = {
+	'.md': CommonMarkParser,
+}
+
+github_doc_root = "https://github.com/JimiC/eveonline-third-party-documentation/tree/Markdown-with-Shpinx/docs/"
```
Comment (Regner) [inline docs/conf.py]: CHange this please :)

Comment (JimiC) [inline docs/conf.py]: Damn. I forgot that, sorry.

**Comment by polarina** [comment]
The package `sphinx_rtd_theme` should be added to `requirements.txt`.

    Theme error:
    sphinx_rtd_theme is no longer a hard dependency since version 1.4.0. Please install it manually.

**Comment by JimiC** [comment]
@polarina Done.

**Pull request merged**